### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by bkuhl**
* Added a list of what's included out of the box with this repo
 * Moved instructions how to get up and running to the CONTRIBUTING.md file

@jlaswell @millidavids

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## FAQ
+
+#### Is there a shortcut for running commands within specific containers?
+
+Yes!  Using the alias below, you can run commands in containers with `dockerexc fpm php -v` instead of `docker exec -it $(docker ps -f name=fpm -q) php -v`.
+
+```
+alias dockerexc='function _docker_exec(){ service=$1; shift; docker exec -it $(docker-compose ps -q ${service}) "$@" };_docker_exec'
+```
```
Comment (jlaswell) [inline README.md]: Maybe mention that users simply need to copy and paste that into their terminal as well?

**Comment by jlaswell** [comment]
Let me know what you think about the comment above, but either way this looks great!

File: README.md (hunk 1 above)
Comment (bkuhl) [inline README.md]: How about "using the alias" links to http://askubuntu.com/a/17538/132639 ?  If we recommend they copy it, they may end up doing that every time and wonder why it stops working "randomly"

Comment (jlaswell) [inline README.md]: That would be a great way to handle that.

**Pull request merged**